include/mysqli.php

```diff
@@ -115,7 +115,15 @@ function db_version() {
 }
 
 function db_timezone() {
-    return db_get_variable('system_time_zone', 'global');
+	$php_tz = ini_get('date.timezone');
+	$php_abbr = new DateTime('now', new DateTimeZone($php_tz));
+	$php_abbr = $php_abbr->format('T');
+	$sql_tz = db_get_variable('system_time_zone', 'global');
+	if ($php_abbr === $sql_tz)
+		return $php_tz;
+
+	echo '<i class="icon icon-warning-sign" title="Mismatch between \'date.timezone\' and \'mysql timezone\'"></i> ';
```

Comment:
This seems to work now as intented. But it seems quite strange ` Europe/London (Interpretiert als Europe/Berlin) `...

It checks there for a config key named `db_timezone`, but I cant find that config in database and dont know how and when this will change.